docs/en/enterprise-edition/rn/prisma-cloud-release-info/features-introduced-in-2024/features-introduced-in-december-2024.adoc

```diff
@@ -0,0 +1,1394 @@
+* `Microsoft.Insights/DiagnosticSettings/Read`
+
+The Reader role includes the above permissions.
+
+
+|*Azure Virtual WAN*
+
+tt:[24.12.1]
+
+//RLP-152956
+
+|*azure-virtual-wan-virtual-hubs*
+
+Additional permission required:
+
+* `Microsoft.Network/virtualHubs/read`
+
+The Reader role includes the above permission.
+
+
+|*Google App Engine*
+
+tt:[24.12.1]
+
+//RLP-152631
+
+|*gcloud-app-engine-service-version*
+
+Additional permissions required:
+
+* `appengine.services.list`
+* `appengine.versions.list`
+
+The Viewer role includes the above permissions.
+
+|*Google App Engine*
+
+tt:[24.12.1]
+
+//RLP-152630
+
+|*gcloud-app-engine-service*
+
+Additional permission required:
+
+* `appengine.services.list`
+
+The Viewer role includes the above permission.
+
+
+|*Google App Engine*
+
+tt:[24.12.1]
+
+//RLP-152628
+
+|*gcloud-app-engine-domain-mapping*
+
+Additional permission required:
+
+* `appengine.applications.get`
+
+The Viewer role includes the above permission.
+
+|*Google Bigquery Data Policy*
+
+tt:[24.12.1]
+
+//RLP-152706
+
+|*gcloud-bigquery-data-policy*
+
+Additional permissions required:
+
+* `bigquery.dataPolicies.list`
+* `bigquery.dataPolicies.getIamPolicy`
+
+The Viewer role includes the above permissions.
+
+
+|*Google Integration Connectors*
+
+tt:[24.12.1]
+
+//RLP-152611
+
+|*gcloud-integration-connectors-endpoint-attachment*
+
+Additional permission required:
+
+* `connectors.endpointAttachments.list`
+
+The Viewer role includes the above permission.
+
+|*Google Integration Connectors*
+
+tt:[24.12.1]
+
+//RLP-151553
+
+|*gcloud-integration-connectors-custom-connector-version*
+
+Additional permissions required:
+
+* `connectors.customConnectors.list`
+* `connectors.customConnectorVersions.list`
+
+The Viewer role includes the above permissions.
+
+|*Google Integration Connectors*
+
+tt:[24.12.1]
+
+//RLP-151552
+
+|*gcloud-integration-connectors-custom-connector*
+
+Additional permission required:
+
+* `connectors.customConnectors.list`
+
+The Viewer role includes the above permission.
+
+|*OCI Vaults*
+
+tt:[24.12.1]
+
+//RLP-149812
+
+|*oci-vault-secret-versions*
+
+Additional permissions required:
+
+* `SECRET_INSPECT`
+* `SECRET_VERSION_INSPECT`
+
+The Reader role includes the above permissions.
+
+|===
+
+[#new-policies]
+=== New Policies
+
+[cols="40%a,60%a"]
+|===
+|*Policies*
+|*Description*
+
+|*Alibaba Cloud VPC flow log not enabled*
+
+tt:[24.12.1]
+//RLP-153196
+
+|This policy identifies Virtual Private Clouds (VPCs) where flow logs are not enabled.
+
+VPC flow logs capture information about the traffic entering and exiting network interfaces in the VPC. Without VPC flow logs, there’s limited visibility into network traffic, making it challenging to detect and investigate suspicious activities, potential data breaches, or security policy violations. Enabling VPC flow logs enhances network monitoring, improves threat detection, and supports compliance requirements.
```

Comment:
In this sentence lets spell out "there is" to remove contractions that create problems with localization.